src/cls/_ZPM/PackageManager/Developer/Lifecycle/Base.cls

```diff
@@ -398,7 +398,7 @@ Method %Reload(ByRef pParams) As %Status
 			Quit
 		}
 		set first = $$$lcase(first)
-    If ($$$ucase($Piece(first,".",*)) '= "MODULE") {
+    If ($$$ucase($Piece(first,".",*)) '= $EXTRACT(##class(%ZPM.PackageManager.Developer.Document.Module).#EXTENSION,2,)) {
```

Comment:
Again should check against both extensions - MODULE and ZPM. Over here and other places in the file